### DISCUSSION TITLE
Boundary Run-Time Check Failure for variable 'module_grib_mp_findgrib_$ITELL'

### DISCUSSION THREAD

**Discussion by avinp07** [NCAR/hrldas#182] [Q&A] [answered]
Hi,

I'm currently in the process of pre-processing ERA5 forcing data. I've managed to extract the forcing and initial fields from the ERA5 data in GRIB format. However, when I attempt to run ./create_forcing.exe namelist.input.ERA5, I encounter an error. Could you please advise on the potential cause of this issue? Is it likely due to a compilation problem or an issue with the data extraction process?

Thank you for your assistance.

**Comment by cenlinhe** [comment, marked as answer]
what is your error message? Did you compile the code successfully?

**Comment by avinp07** [comment]
I have resolved the issue, which was mainly caused by a compilation problem. I made some changes to the user build options while selecting 'Linux with Intel Fortran MPI.' For future reference, here are the compilation options where I have commented few options:
BZIP2_INCLUDE  =       -I/usr/include
BZIP2_LIB      =       -L/usr/lib64 -lbz2
